devfile.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 components:
   - name: tools
     container:
-      image: quay.io/devfile/universal-developer-image:ubi8-38da5c2
+      image: quay.io/okurinny/dashboard.dev:skaffold
```

Comment:
If we are going to support our own image for development we should prepare a separate repository, like this one https://github.com/che-incubator/dash-licenses with a Dockerfile and workflows.

Reply:
done: https://github.com/akurinnoy/dashboard.dev